### PR TITLE
Create Glass.xaml

### PR DESCRIPTION
**Pull request by Rasmagoric**
A full acrylic, seamless with no strokes / borders theme.

image
image

### CODE REVIEW

File: CustomThemesPackage/Glass.xaml [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+                FallbackColor="{ThemeResource SolidBackgroundFillColorBase}"
+                TintColor="{ThemeResource SolidBackgroundFillColorBase}"
+                TintOpacity="0.4" />
+            <AcrylicBrush
+                x:Key="TextControlBackground"
+                BackgroundSource="HostBackdrop"
+                FallbackColor="{ThemeResource SolidBackgroundFillColorQuarternary}"
+                TintColor="{ThemeResource SolidBackgroundFillColorQuarternary}"
+                TintOpacity="0.5" />
+            <AcrylicBrush
```
Comment (yaichenbaum) [inline CustomThemesPackage/Glass.xaml:305]: Should this be a SolidColorBrush that's transparent?

Comment (yaichenbaum) [reply CustomThemesPackage/Glass.xaml]: @Rasmagoric can you do the same to the other brushes (with the exception of the background brushes)?

File: CustomThemesPackage/Glass.xaml [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+            <Color x:Key="SolidBackgroundFillColorTertiary">#020205</Color>
+            <Color x:Key="SolidBackgroundFillColorQuarternary">#020205</Color>
+
+            <Color x:Key="ControlStrokeColorDefault">transparent</Color>
+            <Color x:Key="ControlStrokeColorSecondary">transparent</Color>
+
+            <SolidColorBrush x:Key="TextControlBorderBrush" Color="transparent" />
+
+            <!-- Widget Buttons -->
+            <AcrylicBrush x:Key="ButtonBackground"
```
Comment (yaichenbaum) [inline CustomThemesPackage/Glass.xaml:23]: Can this also be a transparent solid color brush?

**Comment by Rasmagoric** [comment]
The widget backgrounds? It would be seamless into the background as a transparent solid color brush. It looks better differentiated

**Review by yaichenbaum — APPROVED** [review]

**Review by winston-de — APPROVED** [review]
pretty cool

**Pull request merged**